api/src/main/java/org/openmrs/module/registrationcore/RegistrationCoreConstants.java

```diff
@@ -96,4 +96,8 @@
 	public static final String GP_ERROR_HANDLER_IMPLEMENTATION = "registrationcore.errorHandler.implementation";
 
 	public static final String MPI_IDENTIFIER_TYPE_ECID_NAME = "ECID";
-}
\ No newline at end of file
+
+	public static final String LOCAL_FINGERPRINT = "localFingerprint";
```

Comment:
I would change it to LOCAL_FINGERPRINT_NAME or key to emphasize that it doesn't contain some discrete value (see MPI_IDENTIFIER_TYPE_ECID_NAME)

Reply:
Right, I will change it